### PR TITLE
close #9

### LINKED ISSUE #9: see what was removed
would like to see what was removed after it was cleaned, or have an option/command for that

### PR DESCRIPTION
**Pull request by recursive-beast**

### CODE REVIEW

File: Cleaner_42.sh [hunk 1]
```diff
@@ -44,49 +44,88 @@ echo -e "\033[33m\n -- Available Storage Before Cleaning : || $Storage || --\033
...
+	/bin/rm -rf "$@" &>/dev/null
+
+	return 0
+}
+
+function clean {
+	# to avoid printing empty lines
+	# or unnecessarily calling /bin/rm
+	# we resolve unmatched globs as empty strings.
+	shopt -s nullglob
```
Comment (ombhd) [inline Cleaner_42.sh:74]: does shopt work properly on macOS?

File: Cleaner_42.sh [hunk 2]
```diff
@@ -44,49 +44,88 @@ echo -e "\033[33m\n -- Available Storage Before Cleaning : || $Storage || --\033
...
-/bin/rm -rf "$HOME"/Library/Application\ Support/Chromium/Profile\ [0-9]/File\ System &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Default/File\ System &>/dev/null
-/bin/rm -rf "$HOME"/Library/Application\ Support/Google/Chrome/Profile\ [0-9]/File\ System &>/dev/null
-
-#things related to pool (piscine)
-/bin/rm -rf "$HOME"/Desktop/Piscine\ Rules\ *.mp4
-/bin/rm -rf "$HOME"/Desktop/PLAY_ME.webloc
+
+should_log=0
+if [[ "$1" == "-p" || "$1" == "--print" ]]; then
```
Comment (ombhd) [inline Cleaner_42.sh:49]: Can you note --print as well in the README?

**Review by ombhd — CHANGES_REQUESTED** [review]

File: Cleaner_42.sh (hunk 1 above)
Comment (recursive-beast) [reply Cleaner_42.sh:74]: i tested it in one of the clusters iMacs and it works as expected.

File: Cleaner_42.sh (hunk 2 above)
Comment (recursive-beast) [reply Cleaner_42.sh:49]: I'll do that first thing tomorrow.

File: cclean_print.png
[the remark this answers is not in the document]
Comment (recursive-beast) [reply cclean_print.png:1]: I'll do that first thing tomorrow.

File: Cleaner_42.sh (hunk 2 above)
Comment (recursive-beast) [reply Cleaner_42.sh:49]: one thing I hesitated about is using the `getopt` built-in to extract the flag but I ended up using a simple check on the first script argument, should I reconsider implementing it? I think it'll be best for future features/options.

Comment (ombhd) [reply Cleaner_42.sh:49]: I think a simple check would be enough to make it easier for other people to contribute.
Thank you

File: README.md [hunk 3]
```diff
@@ -20,6 +20,10 @@
 1. First : Run `cclean`
 2. Last  : Enjoy ;)
 
+Options:
+
+`-p`, `--path`: print file paths as they are cleaned by the script
```
Comment (ombhd) [inline README.md:25]: Maybe you wanted to write?
```suggestion
`-p`, `--print`: print file paths as they are cleaned by the script
```

File: cclean_print.png
Comment (ombhd) [inline cclean_print.png:1]: 😉

**Review by ombhd — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (recursive-beast) [reply README.md]: sorry about that

**Comment by recursive-beast** [comment]
I guess that sums it up.

**Review by ombhd — APPROVED** [review]

**Comment by ombhd** [comment]
Merged!
Thanks @recursive-beast for your contribution.

**Pull request merged**